contracts/modules/PermissionManager/IPermissionManager.sol

```diff
@@ -9,6 +9,4 @@ interface IPermissionManager {
 
     function changePermission(address _delegate, address _module, bytes32 _perm, bool _valid) external returns(bool);
 
```

Comment:
Please add all the getter function signatures into the `IPermissionManager.sol`.